starwars/src/components/Child.js

```diff
@@ -0,0 +1,24 @@
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from '../App';
+
+export default class Child extends React.Component {
+
+  componentDidMount() {
+    console.log('Child did mount.');
+  }
+
+  value() {
```

Comment:
I'm not sure why you need this method. You are able to just access the props data using `this.props.WHATEVER_YOU_NAMED_YOUR_PROP`.